bitnami/keycloak/21/debian-11/rootfs/opt/bitnami/scripts/libkeycloak.sh

```diff
@@ -72,7 +72,7 @@ keycloak_validate() {
         fi
     fi
 
-    if ! validate_ipv4 "${KEYCLOAK_BIND_ADDRESS}"; then
+    if ! validate_ip "${KEYCLOAK_BIND_ADDRESS}"; then
         if ! is_hostname_resolved "${KEYCLOAK_BIND_ADDRESS}"; then
             print_validation_error print_validation_error "The value for KEYCLOAK_BIND_ADDRESS ($KEYCLOAK_BIND_ADDRESS) should be an IPv4 address or it must be a resolvable hostname"
```

Comment:
Shouldn't the message be updated as well?
```suggestion
            print_validation_error print_validation_error "The value for KEYCLOAK_BIND_ADDRESS ($KEYCLOAK_BIND_ADDRESS) should be an IPv4 or IPv6 address, or it must be a resolvable hostname"
```

Reply:
Yes you're right, thanks for spotting it.